cluster/charts/crossplane/crds/core/v1alpha1/resourceclass.yaml

```diff
@@ -61,7 +61,7 @@ spec:
           type: string
       required:
       - provisioner
-      - providerRef
+      - providerReference
```

Comment:
Why was `providerRef` renamed to `providerReference`?

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#object-references

Reply:
If this was intended, it looks like providerRef -> providerReference changes may have been missed in types.go comments
```console
$ kubectl -n crossplane-system get resourceclass -A
NAMESPACE           NAME               PROVISIONER                                            PROVIDER-REF   RECLAIM-POLICY   AGE
crossplane-system   standard-cluster   gkecluster.compute.gcp.crossplane.io/v1alpha1                         Delete           51m
crossplane-system   standard-mysql     cloudsqlinstance.database.gcp.crossplane.io/v1alpha1                  Delete           51m
```

Reply:
Good catch on the API conventions. We previously had a mix of `fooRef` and `fooReference`, so I renamed these to standardise. I was unaware the API conventions had a position on this and picked the wrong one.  I've just pushed a commit that renames all the `fooReference` fields to `fooRef`, which should also fix the comments.